### PR TITLE
add config option to set output verbosity

### PR DESCRIPTION
**Pull request by Schparky**

### CODE REVIEW

File: domain.go [hunk 1]
```diff
@@ -114,17 +121,23 @@ func getPersonFromList(compareValue string, peopleList []Person) Person {
...
-	caseSensitivityList := getCaseSensitivitySourceAttributeList(attributeMap)
+func personAttributesAreEqual(sp, dp Person, config AppConfig) bool {
+	caseSensitivityList := getCaseSensitivitySourceAttributeList(config.AttributeMap)
+	equal := true
 	for key, val := range sp.Attributes {
 		if !stringsAreEqual(val, dp.Attributes[key], caseSensitivityList[key]) {
-			log.Printf("Attribute %s not equal for user %s. Case Sensitive: %v, Source: %s, Destination: %s \n",
-				key, sp.CompareValue, caseSensitivityList[key], val, dp.Attributes[key])
-			return false
+			if config.Runtime.Verbosity >= 5 {
```
Comment (Baggerone) [inline domain.go]: Should this `5` be a const too?

File: types.go [hunk 2]
```diff
@@ -26,8 +26,14 @@ type DestinationConfig struct {
 	ExtraJSON json.RawMessage
 }
 
+const (
+	VerbosityLow  = 0
+	VerbosityHigh = 10
```
Comment (Baggerone) [inline types.go]: Did you mean to use these somewhere?

Comment (Schparky) [reply types.go]: Originally, but I decided to leave them to document that 0 is low verbosity and 10 is high.

File: domain.go (hunk 1 above)
Comment (Schparky) [reply domain.go]: Good question. Something like `VerbosityMedium`?

**Comment by fillup** [comment]
Looks like this includes "breaking changes" to the API so we should probably do a major version update on this. This will also require going through and updating all existing instances of functions using this package, which we can discuss internally.

**Comment by Schparky** [comment]
> Looks like this includes "breaking changes" to the API so we should probably do a major version update on this. This will also require going through and updating all existing instances of functions using this package, which we can discuss internally.

Good catch. My motivation for making this change was that the existing log output lists only the first changed field on any given user, which could be misleading. But I didn't want to assume that we wanted all fields listed on every instance.

**Pull request merged**